CppCoreGuidelines.md

```diff
@@ -12144,23 +12144,23 @@ References are never owners.
 
 The names are mostly ISO standard-library style (lower case and underscore):
 
-* `T*`			// The `T*` is not an owner, may be `nullptr` (Assumed to be pointing to a single element)
-* `char*`		// A C-style string (a zero-terminated array of characters); can be `nullptr`
-* `const char*`	// A C-style string; can be `nullptr`
-* `T&`			// The `T&` is not an owner, may not be `&(T&)*nullptr` (language rule)
+* `T*`			// The `T*` is not an owner, may be null; assumed to be pointing to a single element.
+* `char*`		// A C-style string (a zero-terminated array of characters); may be null.
+* `const char*` 	// A C-style string; may be null.
+* `T&`			// The `T&` is not an owner, must not be `*static_cast<T*>(nullptr)` (language rule; there are no "null references").
```

Comment:
I agree, and that's why we have this comment.

I'm sorry, I'm not entirely sure what you're aiming at. Are you objecting to the edit? I think with the edit the explanations are both precise and accurate. Is there anything you'd express differently? Note that the original code doesn't work, `*nullptr` doesn't make sense, since `nullptr` isn't a pointer. (That's the motivation for this edit.) Rather, `nullptr` is *convertible* to various pointer types.